catalog/plugin/catalog-plugin-metacard-validation/src/test/java/ddf/catalog/metacard/validation/MetacardValidityMarkerPluginTest.java

```diff
@@ -115,32 +122,40 @@ public void setUp() {
     }
 
     private void verifyCreate(CreateRequest originalRequest, Consumer<Attribute> errorExpectation,
-            Consumer<Attribute> warningExpectation)
+            Consumer<Attribute> warningExpectation, Consumer<Set<String>> expectedTags)
             throws PluginExecutionException, StopProcessingException {
         CreateRequest filteredRequest = plugin.process(originalRequest);
         List<Metacard> filteredMetacards = filteredRequest.getMetacards();
 
-        verifyMetacardErrorsAndWarnings(filteredMetacards, errorExpectation, warningExpectation);
+        verifyMetacardErrorsAndWarnings(filteredMetacards,
+                errorExpectation,
+                warningExpectation,
+                expectedTags);
         verifyRequestPropertiesUnchanged(originalRequest, filteredRequest);
     }
 
     private void verifyUpdate(UpdateRequest originalRequest, Consumer<Attribute> errorExpectation,
-            Consumer<Attribute> warningExpectation)
+            Consumer<Attribute> warningExpectation, Consumer<Set<String>> expectedTags)
             throws PluginExecutionException, StopProcessingException {
         UpdateRequest filteredRequest = plugin.process(originalRequest);
         List<Metacard> filteredMetacards = getUpdatedMetacards(filteredRequest);
 
-        verifyMetacardErrorsAndWarnings(filteredMetacards, errorExpectation, warningExpectation);
+        verifyMetacardErrorsAndWarnings(filteredMetacards,
+                errorExpectation,
+                warningExpectation,
+                expectedTags);
         verifyRequestPropertiesUnchanged(originalRequest, filteredRequest);
     }
 
     private void verifyMetacardErrorsAndWarnings(List<Metacard> filteredMetacards,
-            Consumer<Attribute> errorExpectation, Consumer<Attribute> warningExpectation) {
+            Consumer<Attribute> errorExpectation, Consumer<Attribute> warningExpectation,
+            Consumer<Set<String>> expectedTags) {
```

Comment:
Changed so the methods take a string and made the two tags a constant to pass in.